junebug/channel.py

```diff
@@ -142,3 +152,15 @@ def status(self):
         # TODO: Implement channel status
         status['status'] = {}
         return status
+
+    @inlineCallbacks
+    def send_message(self, amq_client, to_addr, content, **kw):
+        message = TransportUserMessage.send(
+            to_addr=to_addr, content=content, transport_name=self.id)
+        queue = '%s.outbound' % self.id
+        msg = yield amq_client.publish_message(message, routing_key=queue)
+        ret = {}
+        for key, val in msg.payload.iteritems():
+            if key in allowed_message_fields:
+                ret[key] = val
```

Comment:
Would it maybe be worth throwing an error if disallowed fields were given? Maybe a job for jsonschema?